README.md

```diff
@@ -64,18 +65,24 @@ Once everything is ready, `kubectl get service jaeger-query` tells you where to
 
 ### Deploying the agent as sidecar
```

Comment:
No longer deploying as sidecar by default

Reply:
Not by default, but providing instructions on how to deploy the agent as a sidecar is still useful.

Reply:
Ah right.

There should be some note for correct agent discovery from the apps though, as `localhost` (the node) is not available on the pod scope.

Reply:
I'm not quite sure I understand what you mean, but when using a sidecar, `localhost` is indeed correct.